ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -62,6 +66,12 @@ public FastSyncDownloader(
 
   private FastSyncState storeState(final FastSyncState state) {
     fastSyncStateStorage.storeState(state);
+    if (state.getPivotBlockNumber().isPresent()) {
+      trailingPeerRequirements =
```

Comment:
This is a little unexpected in a "storeState" method

Reply:
I was thinking along the lines of storing the new trailing peer requirements was storing state but probably is clearer to just have its own method.